pkg/pool-linear/contracts/LinearPool.sol

```diff
@@ -381,10 +425,12 @@ contract LinearPool is BasePool, IGeneralPool, LinearMath, IRateProvider {
 
         // The full BPT supply will be minted and deposited in the Pool. Note that there is no need to approve the Vault
         // as it already has infinite BPT allowance.
+        uint256 bptAmountOut = _INITIAL_BPT_SUPPLY;
+
         uint256[] memory amountsIn = new uint256[](_TOTAL_TOKENS);
-        amountsIn[_bptIndex] = _MAX_TOKEN_BALANCE;
+        amountsIn[_bptIndex] = _INITIAL_BPT_SUPPLY;
 
-        return (_MAX_TOKEN_BALANCE, amountsIn);
+        return (bptAmountOut, amountsIn);
```

Comment:
Would it save a little to use named return arguments here? Just "bptAmountOut = _INITIAL_BPT_SUPPLY" then. Instead of setting to a constant and immediately returning it.

Maybe the explicit return is clearer/more consistent?

Reply:
Yeah I'm just not a fan of named return arguments in general, but wanted to at least hint at what _MAX_TOKEN_BALANCE stands for here.